qemu/tests/cfg/qemu_iotests.cfg

```diff
@@ -19,13 +19,6 @@
         - qcow2_format:
             only qcow2 qcow2v3
             qemu_io_image_format = qcow2
-        - qed_format:
```

Comment:
This case is for upstream qemu. Please make sure the formats you dropped are deprecated or they won't be used by upstream customers.

Reply:
Thanks for your info. I think upstream still use it for there is still codes update about qed/vdi format from the upstream code. And I will roll back the changes.